src/main/java/nextstep/subway/fare/domain/DiscountRateType.java

```diff
@@ -0,0 +1,36 @@
+package nextstep.subway.fare.domain;
+
+import java.util.Arrays;
+import java.util.function.IntUnaryOperator;
+
+import nextstep.subway.utils.StreamUtils;
+
+public enum DiscountRateType {
+    TODDLER(1, 6, fare -> 0),
+    CHILD(6, 13, fare -> (int) ((fare - 350) * 0.5)),
+    TEENAGER(13, 19, fare -> (int) ((fare - 350) * 0.8)),
+    ADULT(19, Integer.MAX_VALUE, fare -> fare);
+
+    private static final String INVALID_AGE_ERROR_MESSAGE = "나이 정보가 유효하지 않습니다.";
+    private final int minAge;
```

Comment:
```suggestion
    private static final String INVALID_AGE_ERROR_MESSAGE = "나이 정보가 유효하지 않습니다.";
    
    private final int minAge;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가하면 가독성이 상승합니다!

Reply:
구현에 집중하다보니 이런 디테일을 놓쳤네요 ㅎㅎ 잡아주셔서 감사합니다!!👍